### ISSUE TITLE
Use message() (or tidyverse equivalent) to print progress output rather than cat(). (Was: Progress output colors for stergm fit)

### ISSUE THREAD

**Issue by smjenness** [statnet/tergm#2]
I was curious whether the console printing for ergm versus tergm uses a different function, as the ergm part of a stergm estimation is in red while the stergm part is in black. 

<img width="921" alt="screen shot 2018-08-31 at 4 38 02 pm" src="https://user-images.githubusercontent.com/2521570/44934865-72c5aa80-ad3c-11e8-994e-a5157ab86827.png">

These are the default colors in Rstudio. It is not a huge issue, but an inconsistency in output that may confuse some users.

Reproducible example from above:
```r
library(tergm)
nw <- network.initialize(n = 100, directed = FALSE)

fit <- stergm(nw,
              formation = ~edges + concurrent,
              dissolution = ~offset(edges),
              targets = "formation",
              target.stats = c(50, 25),
              offset.coef.diss = log(9),
              estimate = "EGMME",
              eval.loglik = FALSE)
```

**Comment by krivit** [comment]
Ah... I had ported `ergm` to use `message()` when printing progress reports, but I hadn't gotten to doing it for `tergm()`.

**Comment by smjenness** [comment]
That's a helpful explanation, thanks @krivit. 

I will also note that switching to a purely `message()` based approach would also allow for better control of complete suppression of all output from ergm/tergm. 

This is helpful in cases like this:
```r
library(ergm)
nw <- network.initialize(100, directed = FALSE)
fit <- suppressMessages(ergm(nw ~ edges + degree(0), target.stats = c(50, 20)))
```
when depending packages like `EpiModel` run unit tests of ERGMs know to fit well, where the  `testthat` package works more cleanly if there is no console output.

Thanks!

**Comment by chad-klumb** [comment]
Addressed in #11.
